### PR TITLE
Feat/todo app

### PR DESCRIPTION
**Pull request by Amrhub**
#### Components added are being placed in `src/components`
- `Header` component is where I render the header
- `InputTodo` component is responsible for taking input
- `Navbar` component is responsible for displaying the navbar
- `TodoItem` component is responsible for managing each todo item 
- `TodosList` component is responsible for containing all TodoItem
- All of these components are being rendered through the `TodoContainer` component

<small>All components are functional based components using `React Hooks` when needed <small>

#### Styles
All of styles is located at `src/todo.scss` except todoItem styling is located at `src/components/TodoItem.module.scss` as a practice to CSS module styling

#### Pages `src/pages`
- `About` has nested pages inside it `SinglePage` where I rendered `about author` and `about app`
- `NotMatch` is used for pages with no match as the default

#### Deploy
Deployed project using `gh-pages` <br> **[Live Demo](https://amrhub.github.io/react-todo-app/)**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,90 @@
-# react-todo-app
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Todo-app
+
+> Todo-app is a simple todo-list app.
+
+![screenshot](./app_screenshot.png)
+This is just a simple calculator user interface that I built using the [React](https://reactjs.org/) framework.
```
Comment (nicupop729) [inline README.md]: - [x] please update the README file as this is a to-do app and not a calculator

**Review by nicupop729 — COMMENTED** [review]
Good job, @Amrhub

**Review by nicupop729 — APPROVED** [review]
Hi @Amrhub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

## Good Points 👍  
- Descriptive Pull Request
- Hosted project on Github pages is working correctly
- Good presentation of the project
- The menu is working as intended
- No linter errors

I just recommend taking another look at the Readme file, as I wrote as a comment, this is a to-do app and not a calculator.

File: README.md (hunk 1 above)
Comment (Amrhub) [reply README.md]: Opsies, Good observation 🙈

**Pull request merged**